Package.swift

```diff
@@ -375,688 +38,14 @@ let package = Package(
-        // .testTarget(name: "AWSWAFTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWAF", smithyTestUtil], path: "./Tests/Services/AWSWAFTests"),
-        // .testTarget(name: "AWSWAFRegionalTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWAFRegional", smithyTestUtil], path: "./Tests/Services/AWSWAFRegionalTests"),
-        // .testTarget(name: "AWSWAFV2Tests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWAFV2", smithyTestUtil], path: "./Tests/Services/AWSWAFV2Tests"),
-        // .testTarget(name: "AWSWellArchitectedTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWellArchitected", smithyTestUtil], path: "./Tests/Services/AWSWellArchitectedTests"),
-        // .testTarget(name: "AWSWisdomTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWisdom", smithyTestUtil], path: "./Tests/Services/AWSWisdomTests"),
-        // .testTarget(name: "AWSWorkDocsTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkDocs", smithyTestUtil], path: "./Tests/Services/AWSWorkDocsTests"),
-        // .testTarget(name: "AWSWorkLinkTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkLink", smithyTestUtil], path: "./Tests/Services/AWSWorkLinkTests"),
-        // .testTarget(name: "AWSWorkMailTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkMail", smithyTestUtil], path: "./Tests/Services/AWSWorkMailTests"),
-        // .testTarget(name: "AWSWorkMailMessageFlowTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkMailMessageFlow", smithyTestUtil], path: "./Tests/Services/AWSWorkMailMessageFlowTests"),
-        // .testTarget(name: "AWSWorkSpacesTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkSpaces", smithyTestUtil], path: "./Tests/Services/AWSWorkSpacesTests"),
-        // .testTarget(name: "AWSWorkSpacesWebTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSWorkSpacesWeb", smithyTestUtil], path: "./Tests/Services/AWSWorkSpacesWebTests"),
-        // .testTarget(name: "AWSXRayTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSXRay", smithyTestUtil], path: "./Tests/Services/AWSXRayTests"),
+        .testTarget(name: "AWSTranscribeStreamingTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSTranscribeStreaming", smithyTestUtil], path: "./Tests/Services/AWSTranscribeStreamingTests"),
+        .testTarget(name: "AWSTranscribeStreamingIntegrationTests", dependencies: [awsCommonRuntimeKit, clientRuntime, "AWSClientRuntime", "AWSTranscribeStreaming", smithyTestUtil], path: "./IntegrationTests/Services/AWSTranscribeStreamingIntegrationTests", resources: [.process("Resources")]),
```

Comment:
Can we make a clear separation for integration tests with a new line and a mark comment. 

Additionally, we need to figure out what our test plan is for integration tests. Not sure we want them running on every commit to every PR...

Reply:
there should be one